### ISSUE TITLE
[2.x] Update got to v10.x

### ISSUE THREAD

**Issue by XhmikosR** [addyosmani/critical#432]
@bezoerb https://github.com/sindresorhus/got/releases/tag/v10.0.0

We will need to drop support for Node.js 8.x but that's OK IMO for v2.x. From a quick test some tests fail.

**Comment by bezoerb** [comment]
What's the benefit here?

**Comment by bezoerb** [comment]
Currently `got v10` fails with the rejectUnauthorized:false option. 
https://github.com/sindresorhus/got/issues/1148
We can update as  soon as this is fixed or we have a workaround.

Node.js 8.x support is dropped ;)

**Comment by bezoerb** [comment]
Seems this `RequestError: unable to verify the first certificate` issue is caused by the 'old' `pupetteer` version (1.15.0) used in `penthouse`. 

Running the following script works absolutely fine:
```js
const got = require('got');
(async () => {
  try {
    const response = await got(
      'https://fonts.googleapis.com/css?family=Merriweather:300',
      {
        method: 'head'
      }
    );
    console.log('DONE');
  } catch (error) {
    console.error(error);
  }
})();
```

But as soon as `const puppeteer = require("puppeteer");` is  added we get  the mentioned error.
It seems like this error is fixed in puppeteer@2.1.0

**Comment by bezoerb** [comment]
Done with #453
